cfme/tests/cli/test_appliance_update.py

```diff
@@ -121,6 +119,7 @@ def do_yum_update(appliance):
     appliance.wait_for_web_ui()
 
 
+@pytest.mark.meta(coverage=[1714236])
```

Comment:
This should be automates instead ? Since this is automated

Reply:
thanks done